core/templates/dev/head/components/correct_editor_directive.html

```diff
@@ -0,0 +1,38 @@
+<div ng-if="!correctEditorIsOpen"
+title="<[isEditable() ? 'Edit correctness setting' : '']>"
```

Comment:
Indent this and the following line until "title" is left-aligned with "ng-if". Do the same for all continuation lines throughout this file.

(In general, please follow the existing codebase conventions.)

Reply:
Done!